lib/reconcile/vernemq.go

```diff
@@ -292,6 +294,13 @@ func getVerneMQEnvVars(statefulSetName string, cr *apiv1alpha2.Astarte) []v1.Env
 			Value: strconv.Itoa(pointy.IntValue(cr.Spec.VerneMQ.MaxOfflineMessages, 1000000)),
 		})
 
+	// and, starting from Astarte 1.2, add cassandra/scylla env vars
+	c, _ := semver.NewConstraint("> 1.1.999")
```

Comment:
For the future: using `">= 1.2.0-0"` seems more explicit (see https://github.com/Masterminds/semver#working-with-prerelease-versions for the `-0` part)